### PR TITLE
🎨 Default Export

### LINKED ISSUE #68: Change `export` to `export default`
Components:
- [x] Badge.js
- [x] Button.js
- [x] ButtonPlus.js
- [x] Card.js
- [x] HeaderMain.js
- [x] HeaderMenu.js
- [x] HeaderMini.js
- [x] Input.js

Pages:
- [x] Launch.js
- [x] MainMenu.js
- [x] Rides.js

### PR DESCRIPTION
**Pull request by bwnstck**
# Description
Change export to default for components
and changed logo to path instead of internal font

Fixes #68 

## Type of change
- [ x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by arassaria — APPROVED** [review]
LGTM 🚀 👍🏻

File: client/src/pages/AddRide.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-import { Button } from "../components/Button";
-import { addRide } from "../utils/api";
 import { useHistory } from "react-router-dom";
 
+import Badge from "../components/Badge";
+import Card from "../components/Card";
+import HeaderMain from "../components/HeaderMain";
+import Input from "../components/Input";
+import Button from "../components/Button";
+import addRide from "../utils/api";
```
Comment (arassaria) [inline client/src/pages/AddRide.js]: Change `import addRide from "../utils/api"` to `import { addRide } from "../utils/api"`

**Review by arassaria — CHANGES_REQUESTED** [review]
Code breaking

File: client/src/pages/AddRide.js (hunk 1 above)
Comment (bwnstck) [reply client/src/pages/AddRide.js]: already changed ... CI is faster than you are 👩‍🎤

**Review by arassaria — APPROVED** [review]
LGTM 🚀

**Pull request merged**